template-setup.sh

```diff
@@ -3,10 +3,9 @@
 # Add the CI to this repo
 git remote add ci https://github.com/AbstractSDK/rust-ci
 git fetch ci
-git merge ci/main --squash
+git merge ci/main --squash --allow-unrelated-histories
```

Comment:
We can't squash this, it will cause merge conflicts in the future when the CI is updated.